packages/graphql/src/schema/resolvers/subscriptions/subscribe.ts

```diff
@@ -54,12 +54,32 @@ export function generateSubscribeMethod(node: Node, type: "create" | "update" |
 
         const iterable: AsyncIterableIterator<[SubscriptionsEvent]> = on(context.plugin.events, type);
 
-        return filterAsyncIterator<[SubscriptionsEvent]>(iterable, (data) => {
-            return (
-                data[0].typename === node.name &&
-                subscriptionWhere(args.where, data[0], node) &&
-                updateDiffFilter(data[0])
-            );
-        });
+        if (["create", "update", "delete"].includes(type)) {
+            return filterAsyncIterator<[SubscriptionsEvent]>(iterable, (data) => {
+                return (
+                    (data[0] as NodeSubscriptionsEvent).typename === node.name &&
+                    subscriptionWhere(args.where, data[0], node) &&
+                    updateDiffFilter(data[0])
+                );
+            });
+        }
+
+        if (["connect", "disconnect"].includes(type)) {
```

Comment:
Decided to go for a feature-flag in `generate-subscription-types` which blocks the subcription types for connect/ disconnect from appearing in the schema, and always returning false from here in case of a connect/disconnect type of event.